packages/examples/packages/images/tsconfig.json

```diff
@@ -4,7 +4,9 @@
     "baseUrl": "./",
     "paths": {
       "@metamask/*": ["../../../*/src"]
-    }
+    },
+    "module": "ES2020",
+    "moduleResolution": "Node10"
```

Comment:
`snaps-sdk` has a `.ts` file which declares `*.svg`, `*.png`, etc., but this doesn't work in `Node16` or `NodeNext` module resolution. I've kept `snaps-sdk` and the image example at `Node10` for now, to not introduce any breaking changes.